test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -621,15 +580,7 @@ define([
                         // send a job status update; this will trigger an FSM mode change,
                         // which will enable the jobStatus tab
                         await TestUtil.waitForElementChange(
-                            this.kbaseNode.querySelector('[data-button="jobStatus"]'),
-                            () => {
-                                // send a status update
-                                send_STATUS({
-                                    bus: this.bus,
-                                    jobId,
-                                    jobState,
-                                });
-                            }
```

Comment:
move this so it's done in response to the `START_UPDATE` message, as would happen IRL